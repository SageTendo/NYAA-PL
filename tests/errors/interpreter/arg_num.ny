kawaii func() => {

}

uWu_nyaa() => {
    func(1)
}