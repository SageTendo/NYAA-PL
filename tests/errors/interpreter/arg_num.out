Invalid number of arguments provided...